users/signals.py

```
@@ -0,0 +1,15 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.contrib.auth.models import User
+from .models import Profile
```

Ruff (0.7.0)

`.models.Profile` imported but unused

Remove unused import: `.models.Profile`

(F401)